Packs/CommonScripts/ReleaseNotes/1_12_50.md

```diff
@@ -0,0 +1,6 @@
+
+#### Scripts
+
+##### Sleep
+
+- When providing a *seconds* value that is greater than the configured default (5 minutes by default), the sleep will now use a polling task instead of actively sleeping. This change should have no effect on existing content, but will improve the system performance for long sleeps.
```

Comment:
```suggestion
- Improved performance for longer sleep times (5 minutes by default). When providing a *seconds* value that is greater than the configured default (5 minutes by default), the sleep will now use a polling task instead of waiting.
```